src/docs/tune-compression.dox

```diff
@@ -19,6 +19,22 @@ An example of turning on row-store or column-store dictionary compression:
 
 @snippet ex_all.c Configure dictionary compression on
 
+@section raw_compression Raw Compression
```

Comment:
Thanks @daveh86. This is a good starting point, but I'd like to push it further and split file-format compression out into a more obvious section, and describe both the non-raw and raw compression approaches.

I'm a bit torn about whether all of the file format compression description should be in the file_formats_compression section or here.